README.md

```diff
@@ -1 +1,33 @@
-# portfolioProject
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio Project
+
+> Creating my first Portfolio with help from Figma's templates.
+
+
+## Built With
+
+- HTML
+- CSS
+- Git
+
+
+## Authors
+
+👤 **José Fernández-Caminero**
+
+- GitHub: [@fdezcaminero](https://github.com/fdezcaminero)
+- Twitter: [@fdezcaminero](https://twitter.com/fdezcaminero)
+- LinkedIn: [José Fernández-Caminero](https://www.linkedin.com/in/fdezcaminero/)
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
```

Comment:
- [ ] please try to add a section that tells how can one use your project locally

Reply:
Hi @youmari How are you? What do you mean? You mean a Download section? Explaining how to clone the repo and how to use it on a local machine?

Reply:
> Explaining how to clone the repo and how to use it on a local machine?

Exactly! That would be perfect :partying_face: